src/compiler/mcp.md

```diff
@@ -137,14 +137,14 @@ These types of procedural comments can be left on the issue (it's also good to l
 previous section. To facilitate a machine parsable scanning of the concerns please use the following syntax to formally
 register a concern:
 ```
-@rustbot concern reason-for-concern
+@rfcbot concern reason-for-concern
```

Comment:
Question: isn't this only applicable for *FCP*s, does rfcbot (not to be confused with rustbot) handle MCPs now?

Reply:
always has been the case, [AFAICT](https://github.com/rust-lang/triagebot/blob/4b28086e6b7e7566534a8e936ad9b9acf6105a6c/src/github.rs#L2117)

Reply:
Oh cool, did not know that also applied to non-FCP/RFC, i.e. MCP